zcash_client_backend/src/data_api.rs

```diff
@@ -492,6 +492,10 @@ pub trait WalletRead {
     /// will be interpreted as belonging to that account.
     type AccountId: Copy + Debug + Eq + Hash;
 
+    /// Gets some of the account details (e.g. seed fingerprint+index and/or uvk) for a given account id.
+    /// Returns `Ok(None)` if no account by the given ID is known.
```

Comment:
"some" is fuzzy (I immediately wonder "why not all?") and doesn't explain what this method is for or why it should be in the public API. Please expand this documentation.

Also, newline between first and later sentences for rustdoc styling (first sentence appears in the module list view):
```suggestion
    /// Gets some of the account details (e.g. seed fingerprint+index and/or uvk) for a given account id.
    ///
    /// Returns `Ok(None)` if no account with the given ID is known.
```

Reply:
I removed the API.